backends/gdx-backend-robovm/src/com/badlogic/gdx/backends/iosrobovm/objectal/ALWrapper.java

```diff
@@ -0,0 +1,21 @@
+
+package com.badlogic.gdx.backends.iosrobovm.objectal;
+
+import org.robovm.apple.foundation.NSObject;
+import org.robovm.objc.ObjCRuntime;
+import org.robovm.objc.annotation.Method;
+import org.robovm.objc.annotation.NativeClass;
+import org.robovm.rt.bro.annotation.Library;
+import org.robovm.rt.bro.ptr.VoidPtr;
+
+/** @author Jile Gao */
+@Library(Library.INTERNAL)
```

Comment:
This should be `@Library("ObjectAL")`, since ObjectAL is now linked dynamically.
But it is currently wrong in all ObjectAL bindings

Reply:
If it works with `Library.INTERNAL` I would prefer to use a different PR to change this string value.